signing/signer.go

```diff
@@ -98,9 +119,22 @@ func (es *EdSigner) LittleEndian() bool {
 }
```

Comment:
does EdSigner needs this interface?

Reply:
Before my changes it did, but not any more (only `VRFSigner` still needs it). I'll remove it